client/src/components/ModelGraphViewerComponent/ModelGraphViewerCytoscapeComponent/ModelGraphViewerCytoscapeComponent.js

```diff
@@ -216,6 +211,24 @@ export class ModelGraphViewerCytoscapeComponent extends React.Component {
     });
   }
 
+  sessionLayout() {
+    const cy = this.graphControl;
+    const el = cy.nodes("*");
+    const currentLayoutPositions = sessionService.getCurrentModelsLayoutPositions(this.layout);
+    if (Object.keys(currentLayoutPositions).length === 0) {
+      for (let i = 0; i < el.length; i++) {
```

Comment:
I *think* you can do `for (const current of el)`. If not, you can definitely do `el.forEach(current => { ... });`

Reply:
Change complete